noxfile.py

```diff
@@ -92,10 +92,44 @@ def unit(session):
 
 
 @nox.session(python=["2.7", "3.7"])
-def system(session):
-    """Run the system test suite."""
-    system_test_path = os.path.join("tests", "system.py")
-    system_test_folder_path = os.path.join("tests", "system")
+def system_v1beta1(session):
```

Comment:
Is there a specific reason for splitting up the system tests into two sessions? 

If this needs to be made permanent `noxfile.py` should be added to the list of excludes in `synth.py`.

Reply:
I mostly did it to speed up the testing cycle, as it takes a couple minutes to get through system tests for a single endpoint.  I can drop it now that I'm done, but is this common enough we should consider parameterizing or plumbing some form of nox flag?

Reply:
Ah I see. Would you mind opening an issue on https://github.com/googleapis/synthtool/issues to discuss additional parametrizing in the noxfile?

Reply:
Filed https://github.com/googleapis/synthtool/issues/427